worker/games.py

```diff
@@ -149,7 +145,11 @@ def setup_engine(destination, worker_dir, sha, repo_url, concurrency):
         make_cmd = m.read().strip()
       subprocess.check_call(make_cmd, shell=True)
     else:
-      subprocess.check_call(MAKE_CMD + ' -j %s' % (concurrency), shell=True)
+      subprocess.check_call(MAKE_CMD + ' -j %s' % (concurrency) + ' profile-build', shell=True)
```

Comment:
Ok for now. But could you open a issue in order to fix this? I see two possible fixes:
- check that we get correct fail and then pass or
- only avoid stripping after checking which kind of makefile we got

Reply:
I'm not sure what to do here. Do you want to catch specific exceptions (calledProcessError) only?

Reply:
Yes, I would only catch specific exceptions and rethrow all others. By the way, what happens if you `strip` a linux build?

Reply:
then it will lose debug info, which I believe it is even beneficial since we are caching them.

Reply:
I'm a python newbie actually.... wouldn't mind if you did the catching & rethrowing...

Stripping linux binaries works fine, yields some space savings.

Reply:
Ok, let us open an issue with this `try/catch` stuff. I don't know whether I can do it on my linux machine, but it could be that somebody picks it up.